lint/private/patcher.mjs

```diff
@@ -0,0 +1,55 @@
+import childProcess from 'child_process';
```

Comment:
needs some doc like "wrapper around linter tools to create a writeable sandbox of source files, then collect the resulting edits as a patch"